content/SCALE/SCALECLIReference/Service/_index.md

```diff
@@ -3,9 +3,355 @@ title: "Service"
+start service=ssh
+true
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Started Command
+
+The `started` command verifies whether or not a `start` command succeeded for a specified service.
+
+*Possible returns:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true` | The service started successfully. |
+| `false`| The service did not start. |
+{{< /truetable >}}
+
+{{< expand "Verifying a Service Started" "v" >}}
+The `started` command requires you to include the `service` option in the command string. After entering the command correctly, it returns either a true or false output.
+
+From the CLI prompt, enter:
+
+<code>service started service=<i>name</i></code>
+
+From the **service** prompt, enter:
+
+<code>started service=<i>name</i></code>
+
+Where:
+* `name` is the service name. For example, enter ssh for the ssh service.
+
+{{< expand "Command Example" "v" >}}
+```
+started service=ssh
+true
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Started_or_Enabled Command
+
+The `started_or_enabled` command displays whether or not a service will start automatically upon reboot or is running.
+
+*Possible returns:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `true` | The service will restart automatically and/or is running. |
+| `false`| The service is not running, nor will it start automatically. |
+{{< /truetable >}}
+
+{{< expand "Verifying a Service State and Enable Status" "v" >}}
+The `started_or_enabled` command requires you to include the `service` option in the command string. After entering the command correctly, it returns either a true or false output.
+
+From the CLI prompt, enter:
+
+<code>service started_or_enabled service=<i>name</i></code>
+
+From the **service** prompt, enter:
+
+<code>started_or_enabled service=<i>name</i></code>
+
+Where:
+* `name` is the service name. For example, enter ssh for the ssh service.
+
+{{< expand "Command Example" "v" >}}
+```
+started_or_enabled service=ssh
+true
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Stop Command
+
+The `start` command restarts a specified service.
+
+*Possible returns:*
+{{< truetable >}}
+| Property | Description |
+|----------|-------------|
+| `false`| The service stopped or is not running. |
+{{< /truetable >}}
+
+{{< expand "Stopping a Service" "v" >}}
+The `stop` command requires you to include the `service` option in the command string. After entering the command correctly, it returns either a true or false output.
+
+From the CLI prompt, enter:
+
+<code>service stop service=<i>name</i></code>
+
+From the **service** prompt, enter:
+
+<code>stop service=<i>name</i></code>
+
+Where:
+* `name` is the service name. For example, enter ssh for the ssh service.
+
+{{< expand "Command Example" "v" >}}
+```
+stop service=ssh
+false
+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Terminate_Process Command
+
+The `terminate_process` command forces a service to stop and disables it.
+
+*Possible returns:*
```

Comment:
a table is non necessary for one possible result. just make it a sentence. and move this to the details section in the expand.